unitTests/ui/FacetSearchTest.ts

```diff
@@ -88,12 +88,32 @@ export function FacetSearchTest() {
       });
 
       describe('when calling focus', () => {
-        it("should update the accessible element's accessibility properties", () => {
-          const setExpandedFacetSearchAccessibilityAttributes = spyOn(facetSearch, 'setExpandedFacetSearchAccessibilityAttributes');
+        let setExpandedFacetSearchAccessibilityAttributes: jasmine.Spy;
+        beforeEach(() => {
+          setExpandedFacetSearchAccessibilityAttributes = spyOn(facetSearch, 'setExpandedFacetSearchAccessibilityAttributes');
           facetSearch.focus();
+        });
+
+        it("should update the accessible element's accessibility properties", () => {
           expect(setExpandedFacetSearchAccessibilityAttributes).toHaveBeenCalledTimes(1);
           expect(setExpandedFacetSearchAccessibilityAttributes).toHaveBeenCalledWith(facetSearch.facetSearchElement['searchResults']);
         });
+
+        it('should give the "combobox" role to the combobox', () => {
+          expect(facetSearch.facetSearchElement.combobox.getAttribute('role')).toEqual('combobox');
+        });
+
+        it('should give the combobox the aria-owns attribute', () => {
+          expect(facetSearch.facetSearchElement.combobox.getAttribute('aria-owns')).not.toBeNull();
```

Comment:
Instead of asserting not `null`, would it be possible to check that the value is equal to the facet id? Or is the id hard to retrieve?

Same comment for the `aria-controls` in the next test.

Reply:
Sure